backend/src/main/java/com/google/models/Alert.java

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.models;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EmbeddedEntity;
+import com.google.models.Anomaly;
+import com.google.models.Timestamp;
+import java.util.List;
+import java.util.ArrayList;
+import java.time.format.DateTimeParseException;
+
+/** Store alert-related data. */
+public final class Alert {
+  public static final String ALERT_ENTITY_KIND = "alert";
+  public static final String STATUS_PROPERTY = "status";
+  public static final String ANOMALIES_LIST_PROPERTY = "anomaliesList";
```

Comment:
After you provide the createFrom and toEntity functions, these can be hide as private members right?
So does the members in Anomaly.java.

Reply:
`ALERT_ENTITY_KIND` will still need to be `public` because it is needed to query alerts from datastore, which will be written in a servlet outside of this class.

For `STATUS_PROPERTY` and the rest of the properties, I use them when I am writing my tests. If they're not `public` then in my test I will need to duplicate these constants... do you think duplicating these constants in the test and making these private variables will be better?

Personally, I think if just the tests are using them, then it should be okay to make these `private` and duplicate these constants.

Reply:
I see. Then it's fine.